eclipsing-binary-simulator/src/BinarySystemView.jsx

```diff
@@ -679,24 +682,24 @@ export default class BinarySystemView extends React.Component {
         this.initialize(this.initObject);
 
         // This scene offset is necessary for some reason.
-        this.stars.x += 200;
-        this.stars.y += 200;
-        this.orbitalPlane.x += 200;
-        this.orbitalPlane.y += 200;
+        this.stars.x += this.size / 2;
+        this.stars.y += this.size / 2;
+        this.orbitalPlane.x += this.size / 2;
+        this.orbitalPlane.y += this.size / 2;
     }
     componentDidUpdate(prevProps) {
         if (prevProps.star1Temp !== this.props.star1Temp) {
             const star1 = this.stars.getChildByName('star1');
             star1.clear();
             star1.beginFill(getColorFromTemp(this.props.star1Temp));
-            star1.drawCircle(100, 200, 200);
+            star1.drawCircle(0, 0, 200);
```

Comment:
Should `200` be replaced with `this.size / 2`?

Reply:
It could be.. I left this as a "magic number" because it's referred to in the original code like this:
![2019-06-13-152812_820x125_scrot](https://user-images.githubusercontent.com/59292/59461659-e29ab280-8def-11e9-8787-2893808e9e58.png)

This 200 is the star radius. And I'm not sure if it necessarily depends on the scene's width and height.

Reply:
Okay cool. I'll merge as is and you can adjust if needed.